Sources/Subprocess/Errors.swift

```diff
@@ -45,3 +46,34 @@ public enum SubprocessError: Error {
     /// Output string could not be encoded
     case outputStringEncodingError
 }
+
+extension SubprocessError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .exitedWithNonZeroStatus(_, let errorMessage):
+            return "\(errorMessage)"
+        case .unexpectedPropertyListObject:
+            return "The property list object could not be cast to expected type"
```

Comment:
Those two other String values are the plist and json contents when there is an error. I didn't think it would be good to include the full file contents in the error message.

Reply:
Maybe a comment in the code here then explaining why we're not using the associated values.  That way I won't ask again in the future when I forget the reason.  :)

Reply:
done